server/views/pages/tiering-additional-information.njk

```diff
@@ -0,0 +1,555 @@
+                        <li>POs can be responsible for cases of all Tiers (to continuity of relationship) </li>
+                        <li>The Protect axis is more heavily weighted than the Change axis to direct PP resource towards prioritising cases with
+                            the most serious risk of harm. 
+                        </li>
+                        <li> Change axis more heavily weighted for Tiers where Person on Probation s are more likely to demonstrate high responsivity
+                            to undertaking change work 
+                        </li>
+                        <li>Tier A and B cases will need to have the highest resource weighting to ensure public protection requirements are met. </li>
+                        <li>The role boundary between PSO and PO appropriate level work aligns to risk of serious harm to a case and falls at the boundary between Tier C and Tier B.</li>
+                        <li>Tiering calculations should reflect the average annual hours, required by a PP, to meet the policy requirements (e.g. National Standards) and associated mandatory
+                            activity of working on a case. 
+                        </li>
+                        <li>Tier C should have a more equitable split between protecting the public and change work </li>
+                        <li>Tier D with lower risk is more focussed on addressing need and change work </li>
+                        <li>Tiering calculations will occur digitally through specific inputs using set criteria (up to date OASys being encouraged), all associated decisions remain professional
+                            judgement of the SPO (e.g allocations) and probation practitioners (e.g blended supervision arrangements)
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        <br/>
+        <br/>
+
+        <h2 class="govuk-heading-m" id="weight-methodology">Development of weightings methodology</h2>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-full">
+                    <p>The Unified Tiering Model (UTM) requires weightings to feed other organisational tools such as the Workload Measurement Tool (WMT).
+                        (Current Kainos WMT will be replaced by MoJ WMT by Dec 2021)
+                    </p><br/>
+                    <p>Plan is to move beyond dated probation modelling (used in the NPS Interim Tiering Model) and take a new approach using a
+                        renewed activity-based methodology (utilising learning from the OMiC model).
+                    </p><br/>
+                    <strong>2-phase approach to be taken:</strong>
+                    <ul class="govuk-list--bullet">
+                        <li>Phase 1: To facilitate launch of UTM in April 21, will utilise current ITM weightings methodology with additional logic
```

Comment:
Maybe remove this whole section actually - as the work has now been done.

Reply:
Would that be from 441? Or just the part about the 2-phase approach?